deploy/playbooks/roles/common/templates/prod_nodes.py.j2

```diff
@@ -179,6 +179,15 @@ nodes = {
         'labels': ['amd64', 'x86_64', 'centos7', 'huge', 'rebootable'],
         'provider': 'openstack'
     },
+    'ceph-build-xenial': {
+        'script': dedent("""#!/bin/bash
+        curl -u {{ prado_user | default('admin') }}:{{ prado_token }} -L "{{ prado_url }}/setup/slave/?token={{ jenkins_prado_token }}&executors=1&labels=amd64+huge+xfs+xenial+x86_64+rebootable&nodename=ceph-build-xenial__%s" | bash"""),
```

Comment:
You're missing the ``ceph-build-xenial`` label in the 'labels' querystring arg here